### ISSUE TITLE
How to "Deploy live" manually

### ISSUE THREAD

**Issue by BirgitBader** [ePages-de/epages-docs#139]
@choffmeister Please explain how to deploy live in case an issue has to be made available on short notice and cannot wait for being deployed automatically.

**Comment by kayabendroth** [comment]
How long does it take to deploy automatically?

**Comment by BirgitBader** [comment]
Once "develop" is merged into "master, deployment will be done every night at 3 o'clock.

**Comment by choffmeister** [comment]
Since I had nothing to do the last 30 minutes, I adapted the deployment script. The documentation is now deployed with zero downtime. So there is no need anymore, the have the deployment of the live version at 3 o'clock.

The development preview as well as the live version is now deployed every few minutes, in case there is something new to deploy.

**Comment by kayabendroth** [comment]
:+1:

**Comment by BirgitBader** [comment]
awesome, thanks
